src/components/legend/extra.tsx

```diff
@@ -30,7 +30,7 @@ export function renderExtra(extra: string | number, isSeriesHidden?: boolean) {
     'echLegendItem__extra--hidden': isSeriesHidden,
   });
   return (
-    <div className={extraClassNames} title={`${extra}`}>
+    <div className={extraClassNames} title={`${extra}`} aria-label={`${extra}`}>
```

Comment:
`aria-label` doesn't work on elements with no semantic value such as `div` or `span` (unless their role is also changed).

Also, this label just restates the inner text so there's really no need to add it. 

```suggestion
    <div className={extraClassNames} title={`${extra}`}>
```